lib/presentation/profile/profile_route.dart

```diff
@@ -0,0 +1,682 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:flutterapp/presentation/profile/profile_dimens.dart';
+import 'package:flutterapp/resources/app_colors.dart';
+import 'package:flutterapp/resources/app_dimens.dart';
+import 'package:flutterapp/resources/app_images.dart';
+import 'package:flutterapp/resources/app_messages.dart';
+import 'package:flutterapp/resources/app_text_letter_spacing.dart';
+import 'package:flutterapp/resources/app_theme.dart';
+import 'package:flutterapp/widgets/progress/arc_progress.dart';
+import 'package:flutterapp/resources/app_text_sizes.dart';
+import 'package:flutterapp/widgets/progress/linear_progress.dart';
+import 'package:flutterapp/presentation/profile/achievement_widget.dart';
+import 'package:flutterapp/widgets/text/uidisplay_widget.dart';
+import 'package:flutterapp/widgets/text/uitext_widget.dart';
+import 'package:flutter_staggered_grid_view/flutter_staggered_grid_view.dart';
+
+class ProfileRoute extends StatefulWidget {
+  @override
+  State<StatefulWidget> createState() {
+    return _ProfileRouteState();
+  }
+}
+
+class _ProfileRouteState extends State<ProfileRoute> {
+  bool isExpanded = false;
+
+  @override
+  Widget build(BuildContext context) {
+    setStatusBarColor(AppColors.colorBlackGray,
+        statusBarIconBrightness: Brightness.light);
+    return _getProfileWidget();
+  }
+
+  Widget _getProfileWidget() {
+    if (isExpanded) {
+      return _getExpandedWidget();
+    } else {
+      return _getDefaultWidget();
+    }
+  }
+
+  Widget _getDefaultWidget() {
+    return Container(
+      color: AppColors.colorWhiteGray,
+      child: ListView(
+        children: [
+          Container(
```

Comment:
Can you please move this container to a separate widget? Will be much easier to read main items that are on the screen then.